toolchain/sem_ir/typed_insts.h

```diff
@@ -641,6 +641,18 @@ struct InitializeFrom {
   InstId dest_id;
 };
 
+// Given an instruction with a constant value that depends on a generic
+// parameter, selects an instance of that instruction with the constant value
```

Comment:
Is this limited to one generic parameter, or is it generic parameters plural?
```suggestion
// Given an instruction with a constant value that depends on generic
// parameters, selects an instance of that instruction with the constant value
```

Reply:
I'd prefer to keep this as is. I think it's correct: if the value depends on multiple generic parameters, then it depends on a generic parameter (in more than one way), and while I'm not entirely sure why, I find the singular easier to read and think about. We also use the singular [in the design](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/generics/terminology.md#dependent-names).